NOTICE.md

```diff
@@ -34,11 +34,11 @@ The project maintains all source code repositories in the following GitHub organ
 
 This project makes use of the follow third party projects.
 
-golang/go (1.17.2)
+golang/go (1.19.3)
 
 * License: BSD 3-Clause "New" or "Revised" License
 * Project: https://github.com/golang/go
-* Source:  https://github.com/golang/go/releases/tag/go1.17.2
+* Source:  https://github.com/golang/go/releases/tag/go1.19.3
```

Comment:
It will be helpful to all dependencies from the notice file. It seems that `golang.org/x/xerror` and `github.com/google/go-cmp` are not used any more. Double check will help here.